yarn-project/cli/tsconfig.json

```diff
@@ -9,6 +9,9 @@
     {
       "path": "../aztec.js"
     },
+    {
+      "path": "../circuits.js"
+    },
```

Comment:
Is this still needed, given the dependency was removed?

Reply:
My bad, removed in https://github.com/AztecProtocol/aztec-packages/pull/2600/commits/d63e6d57982c2d7aa34480cac542fe275ffb0df4